src/ServiceControl/Infrastructure/SignalR/MessageStreamerConnection.cs

```diff
@@ -39,7 +39,7 @@ protected override async Task OnReceived(IRequest request, string connectionId,
 
                 var message = new OutgoingMessage(Guid.NewGuid().ToString(), new Dictionary<string, string>
                 {
-                    {Headers.EnclosedMessageTypes, messageTypes[messageType]}
+                    {Headers.EnclosedMessageTypes, messageTypes[messageType] }
```

Comment:
Really bizarre that only the closing brace had a warning. Opening brace can be fixed later, if required.